packages/crypto/src/index.ts

```diff
@@ -7,3 +8,5 @@ export * from './Ed25519e';
 export * from './strategies';
 export * from './hexTypes';
 export * from './types';
+
+export const ready = async (): Promise<void> => await sodium.ready;
```

Comment:
```suggestion
export const sodiumReady = async (): Promise<void> => await sodium.ready;
```

Reply:
I would rather hide the specific implementation detail (that we are using sodium underneath), so ready seems more generic

Reply:
I see, ok that makes sense since sodium is the default one